spec/intl.html

```diff
@@ -691,6 +691,218 @@ <h1>GetDateTimeFormatPattern ( _dateStyle_, _timeStyle_, _matcher_, _opt_, _data
     </emu-clause>
     </ins>
 
+    <ins class="block">
+    <emu-clause id="sec-temporal-handledatetimevaluetemporaldate" aoid="HandleDateTimeTemporalDate">
+      <h1>HandleDateTimeTemporalDate ( _dateTimeFormat_, _x_ )</h1>
+
+      <p>
+        The abstract operation HandleDateTimeTemporalDate accepts the arguments _dateTimeFormat_ (which must be an object initialized as a DateTimeFormat) and _x_ (which must be an ECMAScript value has an [[InitializedTemporalDate]] internal slot). It returns a record which contains the appropriate pattern and epochNanoseconds values for the input. This abstract operation functions as follows:
+      </p>
+
+      <emu-alg>
+      1. Assert: _x_ has an [[InitializedTemporalDate]] internal slot.
```

Comment:
In this AO we now know for sure what _x_ is, so it might be clearer to name the variable _temporalDate_ as we do in other AOs. Same for the other AOs below except for HandleDateTimeOthers.